packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.service.js

```diff
@@ -0,0 +1,20 @@
+export default class XdslAccessComfortExchangeService {
+  /* @ngInject */
+  constructor(OvhApiXdsl) {
+    this.OvhApiXdsl = OvhApiXdsl;
+  }
+
+  getOpenedRMAs(xdslId) {
+    return this.OvhApiXdsl.RMA()
+      .v6()
+      .query({
+        xdslId,
+      })
+      .$promise.then((rmas) => {
+        if (rmas.length > 0) {
+          return rmas;
+        }
+        return null;
```

Comment:
i think it's a bad practice to return null if the length is zero. You expect the method to return an empty array if there are no rmas.